### PR TITLE
[testsuite] Add `float_abi` test with `go`

### PR DESCRIPTION
**Pull request by staticfloat**
It appears that go 1.13 does not embed a proper hardfloat flag into the
ELF `e_flags` header.  Let's create a test case where we would expect to
have that flag set, so we can track when this gets fixed.

### CODE REVIEW

File: 0_RootFS/Rootfs/bundled/testsuite/Makefile [hunk 1]
```diff
@@ -1,45 +1,65 @@
...
-	done;
+	# Build a list of `run` targets that we can run serially, in a predictable order
+	@for run_target in $(foreach l,$(LANGUAGES),$(foreach p,$(wildcard $(l)/*),run-project-$(l)-$(notdir $(p)))); do \
+		@$(MAKE) $${run_target};
+	done
+
+.PHONY: build clean run install run-project-% clean-project-%
 
+print-%:
+	@echo "$*=$($*)"
```
Comment (giordano) [inline 0_RootFS/Rootfs/bundled/testsuite/Makefile:65]:
```suggestion
	@echo "$*=$($*)"

```

File: 0_RootFS/Rootfs/bundled/testsuite/Makefile [hunk 2]
```diff
@@ -1,45 +1,65 @@
...
+
+# Add all languages to the overall `build` and `clean` targets
+$(foreach l,$(LANGUAGES),$(eval $(call add_dep,build,build-$(l))))
+$(foreach l,$(LANGUAGES),$(eval $(call add_dep,clean,clean-$(l))))
+$(foreach l,$(LANGUAGES),$(eval $(call add_dep,install,install-$(l))))
 
-build-%:
-	@$(MAKE) -sC $(call unesc,$@)
+build-project-%:
+	@$(MAKE) -sC $(call unesc,$*)
```
Comment (giordano) [inline 0_RootFS/Rootfs/bundled/testsuite/Makefile:45]: Why this change? 🤔

Comment (staticfloat) [reply 0_RootFS/Rootfs/bundled/testsuite/Makefile:45]: Because I want to have language-wide targets like `build-c`, and I don't want it to match `build-%`.
